tests/ttnn/unit_tests/operations/test_creation.py

```diff
@@ -162,6 +162,8 @@ def test_arange(device, start, end, step):
     output_tensor = ttnn.from_device(output_tensor)
     output_tensor = ttnn.to_torch(output_tensor)
     output_tensor = output_tensor[-1, -1, -1, :]
+    if (round((end - start) / step + 0.4)) % 2 != 0:
+        output_tensor = output_tensor[:-1]
```

Comment:
Can you explain this calculation?

Is this checking that if we expect an odd number of elements, then we drop the last padded element?

Why do we not just do the same calculation as done in the op? We have a `divup` util function

`divup((end - start), step) % 2 != 0`

Reply:
Hi @tt-aho , previous calculation is to find the ceil value of the number, but for that we need to import math function, so instead of importing new package, i just used round function to do the same, now i have updated to do like they did in `div_up` function `if int(((end - start) + step - 1) / step) % 2 != 0`

Reply:
Why are we avoiding using the divup util function and replicating the calculation locally?

You can import it from tests/ttnn/utils_for_testing.py models/utility_functions.py

Reply:
Thanks for pointing the `divup` location, i thought `div_up` is in `math.hpp` so cant imported in py file. Now i have updated kindly review it once